tests/vibe.web.rest.1140/source/app.d

```diff
@@ -0,0 +1,52 @@
+import std.datetime;
+import vibe.d;
+
+interface IOrientDBRoot
+{
+	@property IOrientDBQuery query();
+}
+
+interface IOrientDBQuery
+{
+	@method(HTTPMethod.GET)
+	@path(":db_name/sql/:query/:result_set_size")
+	Json sql(string _db_name, string _query, int _result_set_size);
+}
+
+class OrientDBRoot : IOrientDBRoot {
+	private OrientDBQuery m_query;
+	override @property IOrientDBQuery query() { return m_query; }
+	public this() { this.m_query = new OrientDBQuery(); }
+}
+
+class OrientDBQuery : IOrientDBQuery {
+	override Json sql(string _db_name, string _query, int _result_set_size) {
+		assert(_db_name == Param1, _db_name);
+		assert(_query == Param2, _query);
+		assert(_result_set_size == Param3, to!string(_result_set_size));
+		return Json.emptyObject;
+	}
+}
+
+enum Param1 = "twitter_data";
+enum Param2 = "select DownloadedDateTime from Message";
```

Comment:
*Shoots bullet*
Gah, I see. I don't like the idea of undoing work because we've done it too early. In addition, removing this assert is a Bad Idea, since it wouldn't catch other bugs.
I probably need to sleep over this, thanks for reviewing :)